agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/VariantRepository.java

```diff
@@ -60,14 +62,19 @@ public Variant getVariant(String variantID) {
 		query += " WHERE variant.primaryKey = $" + paramName;
 		query += " OPTIONAL MATCH consequence=(:GenomicLocation)--(variant:Variant)-[:ASSOCIATION]->(:TranscriptLevelConsequence)"
 				+ "<-[:ASSOCIATION]-(t:Transcript)<-[:TRANSCRIPT_TYPE]-(:SOTerm)";
-		query += " OPTIONAL MATCH gene=(t:Transcript)-[:TRANSCRIPT]-(:Gene)--(:GenomicLocation)";
+		query += " OPTIONAL MATCH gene=(t:Transcript)-[:TRANSCRIPT]-(ge:Gene)--(:GenomicLocation)";
+		query += " OPTIONAL MATCH geneSpecies=(ge:Gene)--(:Species)";
 		query += " OPTIONAL MATCH transcriptLocation=(t:Transcript)-[:ASSOCIATION]-(:GenomicLocation)";
 		query += " OPTIONAL MATCH exons=(:GenomicLocation)--(:Exon)-[:EXON]->(t:Transcript)";
-		query += " RETURN p1, consequence, gene, exons, transcriptLocation ";
+		query += " RETURN p1, consequence, gene, geneSpecies, exons, transcriptLocation ";
 
 		Iterable<Variant> variants = query(query, map);
 		for (Variant a : variants) {
 			if (a.getPrimaryKey().equals(variantID)) {
+				List<Transcript> list = a.getTranscriptList();
+				if(CollectionUtils.isNotEmpty(list)){
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'{' is not preceded with whitespace.